mars-rover-new/src/mars_rover_new/core.clj

```diff
@@ -1,7 +1,94 @@
+;; Mars Rover Exercise
+
 (ns mars-rover-new.core
+  (:require [clojure.string :as string])
   (:gen-class))
 
+;; todo: input validation, check invalid moves
+
+;; input validation: check for number of args, catch errors during type conversion
+
+(declare add-plateau-size, add-rover-position, add-cmd-sequence)
+
+(def info-map {:plateau-size {:x nil
```

Comment:
Consider renaming this. It's not very clear how it's being used at a first glance.

Reply:
This is a map to store plateau size, rover position, and command sequence. All of these come from the user. 

So this map has plateau size, which is info about Mars, and rover position + commands, which is info about the rover.

What do you think of:
user-params
mars-rover-params
mars-rover-data